k8s-daemonset/k8s/api.yml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: api
+spec:
+  replicas: 3
+  selector:
+    app: api
+  template:
+    metadata:
+      labels:
+        app: api
+    spec:
+      dnsPolicy: ClusterFirst
+      containers:
+      - name: service
+        image: buoyantio/helloworld:test7
```

Comment:
Cool, let's push your changes to the app as 0.0.4, and change all of the references (in this file and in the hello-world*.yml files) to use the 0.0.4 image.